components/global/Header/Navigation.js

```diff
@@ -42,15 +38,17 @@ export default function Navigation({
     }
     if (navList?.current?.contains(e.target)) return;
     setActive(null);
+    if (desktopSetter) desktopSetter(false);
```

Comment:
Instead of `useEffect` in response to `active` changing, now just calling parent state setter (`desktopSetter`) where `setActive` is called.  Note, it is not called everywhere `setActive` is called, only when the value of `active` will impact the on-screen visibility of the desktop nav.